docs/export.md

```diff
@@ -1,8 +1,14 @@
 The [automatic export process](https://chromium.googlesource.com/chromium/src/+/master/docs/testing/web_platform_tests.md#automatic-export-process) tries to continuously export changes from [blink/web_tests/external/wpt](https://cs.chromium.org/chromium/src/third_party/blink/web_tests/external/wpt/) into [wpt](https://github.com/web-platform-tests/wpt).
 
-If the exporter isn't running, please [file an exporter bug](https://bugs.chromium.org/p/chromium/issues/entry?components=Blink%3EInfra%3EEcosystem&cc=weizhong@google.com&summary=[WPT%20Export]).
+If the exporter isn't running, please [file an exporter bug](https://bugs.chromium.org/p/chromium/issues/entry?components=Blink%3EInfra&summary=[WPT%20Export]).
 
-When an export run has failed with "The following commit did not apply cleanly", it's often because the CL builds on another which hasn't yet been exported to wpt, so if there are blocked export PRs, handle those first. If a Chromium CL has been reverted but original CL hasn't been successfully exported to wpt yet, then both can be skipped by closing the original export PR and changing its description, see [example PR](https://github.com/web-platform-tests/wpt/pull/11357).
+When an export run has failed with "The following commit did not apply cleanly", it's often because the CL builds on another which hasn't yet been exported to wpt, so if there are blocked export PRs, handle those first.
+If a Chromium CL has been reverted but original CL hasn't been successfully exported to wpt yet, then both can be skipped by:
+1. Closing the original export PR.
+1. Adding the revert CL's [`Change-Id` footer][change-id] to the original PR's description.
```

Comment:
Do you have an example CL that has this?

Reply:
Added link to https://github.com/web-platform-tests/wpt/pull/53146